build.gradle

```diff
@@ -92,6 +112,22 @@ task apiGateway(type: Test) {
   classpath = sourceSets.apiGatewayTest.runtimeClasspath
 }
 
+task runConsumerPactTests(type: Test) {
+  description = "Runs Contract Tests"
+  testClassesDirs = sourceSets.testContract.output.classesDirs
+  classpath = sourceSets.testContract.runtimeClasspath
+  useJUnitPlatform()
+}
+
+pact {
+  publish {
+    pactDirectory = 'target/pacts'
+    pactBrokerUrl = System.getenv("PACT_BROKER_FULL_URL") ?:'http://localhost:80'
```

Comment:
missing space

Reply:
looks like this env var is never set in tf

Reply:
As of now not yet configured in the pipeline.